datatypes.go

```diff
@@ -873,3 +873,13 @@ func SetMaintenanceUserAndTime(maintenanceCheck MaintenanceCheck, maintenance in
 	}
 	maintenanceCheck.SetMaintenance(&maintenanceInfo, maintenance)
 }
+
+type SchedulerParams struct {
+	Event             NotificationEvent
+	Trigger           TriggerData
+	Contact           ContactData
+	Plotting          PlottingData
+	ThrottledOld      bool
+	SendFail          int
+	ReschedulingDelay time.Duration
```

Comment:
`ReschedulingDelay` imho would leave this parameter inside the config for the `StandardScheduler` structure as it does not change unlike other parameters, @Tetrergeru what do you think?

Reply:
Done